kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/installation/kroxylicious/CertManager.java

```diff
@@ -55,7 +58,12 @@ public void deploy() {
      * @throws IOException the io exception
      */
     public void delete() throws IOException {
+        if (Boolean.parseBoolean(Environment.CERT_MANAGER_INSTALLED)) {
```

Comment:
The logic looks weird...  In deploy you decide based on namespace OR env var.  In delete you only decide based on env var.